src/lang/en.json

```diff
@@ -1027,5 +1027,7 @@
     "greater than": "greater than",
     "less than or equal to": "less than or equal to",
     "greater than or equal to": "greater than or equal to",
-    "record": "record"
+    "record": "record",
+    "elksUsername": "Username",
+    "elksPassword": "Password"
```

Comment:
For short things, please prefer to just use the word you need as a translation key.
This reduces the amount of work our translators have to do